### PR TITLE
put acbuild in sdk

### PR DESCRIPTION
**Pull request by mischief**
depends on coreos/manifest#83

might pull this into dev/prod images too so ACIs can be built on CoreOS.

### CODE REVIEW

File: app-emulation/acbuild/acbuild-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+SLOT="0"
+IUSE=""
+
+DEPEND=">=dev-lang/go-1.4.3"
+RDEPEND="sys-apps/kmod
+		app-crypt/gnupg
+		sys-apps/systemd"
+
+src_compile(){
+		./build
```
Comment (marineam) [inline app-emulation/acbuild/acbuild-9999.ebuild]: Indentation should be a single tab.

Is there anything special about this build? Can it use our go eclass?

**Comment by marineam** [comment]
Go ahead and add this to the manifest as well so the git repo is fetched in advance.

**Comment by marineam** [comment]
Ping? iirc @vcaputo was interested in this too

File: app-emulation/acbuild/acbuild-9999.ebuild (hunk 1 above)
Comment (mischief) [inline app-emulation/acbuild/acbuild-9999.ebuild]: sorry, something seems terribly wrong with my vim in the sdk..

**Comment by mischief** [comment]
@marineam fixed indents.

can't use coreos-go since it won't set version, the binary will end up with https://github.com/appc/acbuild/blob/master/lib/version.go#L19

ptal.

**Comment by mischief** [comment]
cc @dgonyeo

File: app-emulation/acbuild/acbuild-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+DESCRIPTION="A build tool for ACIs"
+HOMEPAGE="https://github.com/appc/acbuild"
+SRC_URI=""
+
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE=""
+
+DEPEND=">=dev-lang/go-1.4.3"
```
Comment (dgonyeo) [inline app-emulation/acbuild/acbuild-9999.ebuild]: Is CoreOS still on go 1.4? It's getting kinda old by now.

Comment (marineam) [inline app-emulation/acbuild/acbuild-9999.ebuild]: Yes, won't be moving to 1.5 until we get docker 1.10

**Comment by mischief** [comment]
@marineam ptal, using coreos-go now.

File: app-emulation/acbuild/acbuild-9999.ebuild [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+RDEPEND="sys-apps/kmod
+	app-crypt/gnupg
+	sys-apps/systemd"
+
+src_compile(){
+	if [[ "${PV}" == 9999 ]]; then
+		local v=$(git describe --long --dirty); v=${v#v}; v=${v/-/+};
```
Comment (marineam) [inline app-emulation/acbuild/acbuild-9999.ebuild]: use `local v` `v=$(...) || die` so errors are not ignored.

**Comment by marineam** [comment]
LGTM

**Comment by mischief** [comment]
can update this ebuild later if the coreos-semver.eclass gets in.

**Pull request merged**